libs/designsystem/modal/src/modal-wrapper/config/modal-config.ts

```diff
@@ -15,11 +20,13 @@ export interface ModalConfig {
   siblingModalRouteActivated$?: Observable<ActivatedRoute>;
   flavor?: ModalFlavor;
   componentProps?: { [key: string]: any };
+  showAlert?: ShowAlertCallback;
```

Comment:
When the callback is set on the parent modal from the embedded component we call it `canDismiss` like Ionic, but here in the `ModalConfig` we call it `showAlert`. Wouldn't it be better to keep the naming consistent?

Reply:
I concur!